core/dbt/config/project.py

```diff
@@ -298,7 +298,7 @@ def render_package_metadata(self, renderer: PackageRenderer) -> ProjectPackageMe
             raise DbtProjectError("Package dbt_project.yml must have a name!")
         return ProjectPackageMetadata(self.project_name, packages_config.packages)
 
-    def check_config_path(self, project_dict, deprecated_path, exp_path):
+    def check_config_path(self, project_dict, deprecated_path, exp_path=None, default=None):
```

Comment:
I don't know what `exp_path` is.....expected path? Explicit path? Something else? An explicit variable would be nice for readable/reasoning.

Reply:
Also what's `default` exactly?

Reply:
I'll rename these!
- `exp_path` &rarr; `renamed_path`
- `default` &rarr; `default_value`